### PR TITLE
User can see prompt for current letter.

### PR DESCRIPTION
**Pull request by ChrisJVoss**
screen shot 2017-06-09 at 2 25 59 pm

### CODE REVIEW

**Review by victormejia — CHANGES_REQUESTED** [review]
While this works, you want to do this initial highlight of the first letter in JavaScript. To do so, you will probably need to render the text as `spans`s

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Please re-title the Pull Request to explain the user value. It's generally acceptable to just re-use the Issue title.

File: main.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-unused-vars */
+
+var text = "Lorem ipsum dolor sit amet, consectetur adipiscing elit."
+var splitText = text.split('')
+var $span = document.querySelector('#text')
+
+for (var i = 0; i < splitText.length; i++) {
+  var $text = document.createElement('span')
+  $text.textContent = splitText[i]
+  $text.className = ('id')
```
Comment (victormejia) [inline main.js]: a class name of `id` is a bit awkward. Perhaps give a class name of `character`

**Review by victormejia — CHANGES_REQUESTED** [review]

File: main.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-unused-vars */
+
+var text = "Lorem ipsum dolor sit amet, consectetur adipiscing elit."
+var splitText = text.split('')
+var $span = document.querySelector('#text')
+
+for (var i = 0; i < splitText.length; i++) {
+  var $text = document.createElement('span')
+  $text.textContent = splitText[i]
+  $text.className = ('character')
```
Comment (victormejia) [inline main.js]: no need to set this to `('character')`, just `'character'`

**Review by victormejia — APPROVED** [review]

**Pull request merged**